example/CMakeLists.txt

```diff
@@ -58,19 +63,16 @@ FOREACH (source_file ${exec_PROGRAMS})
     TARGET_LINK_LIBRARIES (${source_file_we} Boost::program_options)
 ENDFOREACH ()
 
-FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../example/broker.conf DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/broker.conf DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../test/certs/mosquitto.org.crt DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../test/certs/server.crt.pem DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../test/certs/server.key.pem DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../test/certs/cacert.pem DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
 IF ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+   FILE(COPY ${CMAKE_CURRENT_SOURCE_DIR}/broker.conf DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/Release)
```

Comment:
I noticed that this doesn't support debug build. The problem is originally exists.
But if we get the current build type (Debug or Release), then the code would become better.

Reply:
Yes. Also one problem i have with this. It is a separate target. If you build only broker, the conf files are not updated. And also, i wanted to name the key for broker broker.key.pem instead of server.key.pem

Reply:
What do you mean?
If you update the cert file, then you need to re-run cmake explicitly otherwise the cert file is not updated?
I think that it is natural for cmake.
Perhaps you can add your own dependency. But it is too difficult for me.